.storybook/main.ts

```diff
@@ -0,0 +1,20 @@
+import type { StorybookConfig } from "@storybook/react-webpack5";
+
+const config: StorybookConfig = {
+  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(ts|tsx)"],
```

Comment:
Looking at the storybook docs, it seems like mdx stories aren't supported anymore. How does this work?

Maybe this isn't telling Storybook "this is an mdx file, parse it as such" but instead just telling it "this is a story"?

Reply:
Yeah I can see how this might be confusing. Check out [Storybook Docs / MDX / Setup custom documentation](https://storybook.js.org/docs/writing-docs/mdx#setup-custom-documentation). 

The nuance here, as far as I can tell, is that Storybook no longer supports using MDX to write a completely self-contained story. You can still use MDX but you have to use it in conjunction with Storybook's Component Story Format (CSF).

So you can use MDX + CSF to create richer Stories. But also, and here's the second nuanced bit, you can write MDX files that don't include CSF stories, which can serve as standalone doc pages.